src/js/components/Chart/Chart.js

```diff
@@ -280,6 +282,7 @@ const Chart = React.forwardRef(
               <title>{label}</title>
               <path
                 d={d}
+                fill={patternId ? `url(#${patternId})` : undefined}
```

Comment:
Interesting, according to docs I was under the impression that `pattern` is only relevant for `area` and not for bars.

Reply:
yep, fixed